UserExperience_Remove_CssJs_Page_View.js

```diff
@@ -11,14 +11,19 @@ jQuery(function() {
 		'click',
 		'#w3tc_remove_cssjs_singles_add',
 		function() {
-			var single_path = prompt('Enter CSS/JS URL.');
+			var singlePath = prompt('Enter CSS/JS URL.');
 
-			if (single_path !== null) {
+			if (singlePath === null) {
```

Comment:
Use Yoda conditions and spacing -- `if ( null === singlePath ) {`.

Reply:
Yoda applied